src/ORM/Hierarchy/Hierarchy.php

```diff
@@ -770,7 +770,7 @@ public function numChildren($cache = true)
      */
     public function showingCMSTree()
     {
-        if (!Controller::has_curr()) {
+        if (!Controller::has_curr() || !class_exists(LeftAndMain::class)) {
```

Comment:
Heh I didn't know you could do `class_exists(UnknownClass::class)` without a fatal error, but apparently that works